### DISCUSSION TITLE
Mikrotik IKEv1 and strongswan roadwarrior

### DISCUSSION THREAD

**Discussion by martinprejsa** [strongswan/strongswan#2372] [Q&A] [answered]
I have a Mikrotik router which has IKEv1 ipsec configured. I'd like the client to automatically receive remote_ts which is defined in the mode config. As far as i am aware this should work, because Mikrotik is using Cisco's Unity plugin to send split-include attributes to the client. Yet i haven't been able to make it work. Is it even possible?

**Comment by tobiasbrunner** [comment]
If it actually uses the Cisco Unity extensions, probably.

**Reply by martinprejsa** [reply]
It says so in the [documentation](https://wiki.mikrotik.com/wiki/Manual:IP/IPsec#Mode_configs), is there any way to check if this is correct?

**Comment by martinprejsa** [comment, marked as answer]
It does indeed work, i forgot to enable the option to send cisco vendor id payloads to the server. [Documentation](https://docs.strongswan.org/docs/5.9/plugins/unity.html#_configuration)
